src/main/webapp/css/style-sign-in.css

```diff
@@ -0,0 +1,57 @@
+#sign-in-box {
+  border-radius: 5px;
+  border: 1px solid grey; 
+  box-sizing: border-box;
+  height: 64px;
+  padding: 8px;
+  width: 210px;
+}
+
+#profile-photo-container {
+  float: left;
+  height: 48px;
+  padding-right: 8px;
+  vertical-align: middle;
+  width: 48px;
+}
+
+#profile-photo {
+  border-radius: 50%;
+  height: 100%;
+  object-fit: cover;
+  vertical-align: middle;
+  width: 100%;
+}
+
+#g-signin-container {
+  align-items: center;
+  display: flex;
+  height: 100%;
+}
+
+.dropdown {
+  cursor: pointer;
+  display: inline-block;
+  position: relative;
+}
+
+.dropdown-content {
+  background-color: #f9f9f9;
+  box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
```

Comment:
Super nit: if you're using zero pixels (for something other than the `flex` property), you don't need the `px`. https://google.github.io/styleguide/htmlcssguide.html#0_and_Units

Also please add some spaces to your rgba, e.g. `rgba(0, 0, 0, 0.2)`.